refresh/templates/services/pushnotifications/initializeBluemixService.swift

```diff
@@ -0,0 +1,6 @@
+
+    let pushNotificationsService = try manager.getPushSDKService(name: "<%- service.name %>")
+
+    let region = "<%- service.region %>".isEmpty ? PushNotifications.Region.US_SOUTH : "<%- service.region %>"
+
+    pushNotifications = PushNotifications(service: pushNotificationsService, region: region)
```

Comment:
I did it this way so that I could use the constant defined in PushNotifications: `PushNotifications.Region.US_SOUTH` instead of hardcoding `us_south`. Your suggestion would certainly clean up the code a bit.